db/primary_migrate/20221201143826_alter_irs_attempt_api_log_files.rb

```diff
@@ -0,0 +1,6 @@
+class AlterIrsAttemptApiLogFiles < ActiveRecord::Migration[7.0]
+  def change
```

Comment:
So rails has a thing where migrations run with `change` can be reversed. I’m not sure how these two statements work when reversed so I would consider breaking them into two separate migrations. Or, defining separate `up` and `down` methods instead of `change`